app/frontend/src/aquifers/components/Documents.vue

```diff
@@ -18,14 +18,50 @@
           <ul v-else-if="files && files.public && files.public.length">
             <li v-for="(file, index) in files.public" :key="index">
               <a :href="file.url" :download="file.name" target="_blank">{{file.name}}</a>
+              <a class="fa fa-trash fa-lg"
+                variant="primary"
+                v-if="editMode"
+                style="margin-left: .5em"
+                href="#"
+                v-on:click="confirmDeleteFile(file.name, 'public', $event)" />
             </li>
           </ul>
           <div v-else>
               No additional documentation currently available for this aquifer.
           </div>
         </div>
+        <div class="row no-gutters">
+          <div class="col-md-12">
+            <h4>Internal documentation - authorized access only</h4>
+            <div v-if="error">
+              {{error}}
+            </div>
+            <ul v-else-if="files && files.private && files.private.length">
+              <li v-for="(file, index) in files.private" :key="index">
+                <a :href="file.url" :download="file.name" target="_blank">{{file.name}}</a>
+                <a class="fa fa-trash fa-lg"
+                  variant="primary"
+                  v-if="editMode"
+                  style="margin-left: .5em"
```

Comment:
You can add class `ml-1` (or a higher value) for increasing the left margin instead of using an inline style.